tree/20_cfe_basics/packages.cf

```diff
@@ -16,16 +16,7 @@
 #
 #####################################################################################
 
-# Used in Rudder tests
-body package_module test
-{
-      query_installed_ifelapsed => "$(ncf_def.package_module_query_installed_ifelapsed)";
-      query_updates_ifelapsed => "$(ncf_def.package_module_query_updates_ifelapsed)";
-      #default_options =>  {};
-}
-
-# Not upstream
-body package_module zypper_pattern
+body package_module ncf_module
```

Comment:
IIRC this configures the "ncf_module" package module which doesn't exist. The name must match an existing package module.